message/internal_msg_builder.go

```diff
@@ -520,6 +605,10 @@ func InternalGossipRequest(
 
 type Timeout struct{}
 
+func (_ Timeout) String() string {
```

Comment:
super nit, but typically we do:
```suggestion
func (Timeout) String() string {
```
rather than using the `_`